sonar/modules/documents/jsonschemas/documents/document-v1.0.0_src.json

```diff
@@ -81,36 +78,39 @@
           },
           "order": {
             "title": "Position",
-            "description": "Position of the file",
+            "description": "Position of the file, The lowest position means file is the main file.",
             "type": "integer",
             "default": 1,
             "minimum": 1
           },
           "external_url": {
             "title": "URL to file",
-            "description": "URL to file hosted in an external platform.",
             "type": "string",
             "minLength": 1,
-            "pattern": "^https?://"
+            "pattern": "^https?://.*"
```

Comment:
this can be replaced by `format="uri"`: https://json-schema.org/understanding-json-schema/reference/string.html#uri-template

Reply:
with `format: uri` the form is not validated, but anyway I will use format + pattern.